src/routes/utils/ExampleWrapper.svelte

```diff
@@ -8,6 +8,7 @@
   import type { PageData } from '../$types';
   import { identity } from 'svelte/internal';
```

Comment:
@shinokada this import is not used, is it necessary?

Reply:
You can remove `identity`, but `PageData` is used in line 26.